src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -552,6 +550,11 @@ public void Insert(int index, Page page)
             navigationPages.Insert(index, page);
             Add(page);
             page.Navigator = this;
+            if (index == PageCount-1)
```

Comment:
Probably there should be space as below.
'PageCount-1' -> 'PageCount - 1'

Reply:
Of course :).